docs/crd_mysql_dump.md

```diff
@@ -0,0 +1,25 @@
+# Dump
+
+`Dump` is a custom resource definition (CRD) that represents
+the location of a MySQL dump file.
+
+| Field        | Type                      | Description                                  |
+| ------------ | ------------------------- | -------------------------------------------- |
+| `apiVersion` | string                    | APIVersion.                                  |
+| `kind`       | string                    | Kind.                                        |
+| `metadata`   | [ObjectMeta]              | Standard object's metadata.                  |
+| `spec`       | [DumpSpec](#DumpSpec)     | Specification of desired state of full dump. |
+| `status`     | [DumpStatus](#DumpStatus) | Most recently observed status of full dump.  |
```

Comment:
There is no `status` now.

Reply:
Combined Dump with Binlog and removed status.